### ISSUE TITLE
Hi, how did you train your model on the MPII dataset?

### ISSUE THREAD

**Issue by mks0601** [ZheC/Realtime_Multi-Person_Pose_Estimation#161]
Hi, I`m gyeongsik moon from computer vision lab, seoul national university.

I try to train my model on the MPII dataset, however, many of papers did not mention the details in training. There are single person frames and multi person frames in the MPII dataset. In multi person track, only multi person frames (which include 'group') are used in testing. Then, did you only use multi person frames in training? Or both of single and multi person frames are used?

**Comment by ZheC** [comment]
Yes, I think I am using both single and multi person frames. I check the code, I was loading the mpii_human_pose_v1_u12_1.mat and it should contain all the images. 

Please let me know any other question you have. I also find the json file to generate the lmdb for the MPII dataset. It contains all the images name and annotations, let me know if you need it to double check.

**Comment by yw155** [comment]
@ZheC, thanks for your great work. I am trying to train a model on MPII dataset but I cannot get an ideal result. So I would like to ask you that what the super-parameters were used in training the MPII dataset, such as learning rate, mini-batch, the number of GPU, epoch number, data augmentation parameters, etc. In addition, is it possible to upload the lmdb file of MPII dataset. Thanks in advance.
